### DISCUSSION TITLE
License suspended error from Antmedia

### DISCUSSION THREAD

**Discussion by MBFLYT** [ant-media/Ant-Media-Server#5894] [FAQs]
{
    "definition": "license_suspended_please_renew_license",
    "command": "error"
}

Hey @USAMAWIZARD, 

We have been facing this issue from yesterday, and it will be highly appreciable if this can be solved ASAP.

**Comment by Mohit-3196** [comment]
Hi @MBFLYT,
This error shows that your license has not been renewed and it has expired.
Can you please share your license key with the solutions team at contact@antmedia.io so that we can check it further?

Thank you.

**Reply by MBFLYT** [reply]
Than you for the quick response @USAMAWIZARD !!

**Reply by Mohit-3196** [reply]
Hi @MBFLYT 
This is Mohit :)

Did you reach out to contact@antmedia.io ?

**Reply by MBFLYT** [reply]
Hey @Mohit-3196, Yeah my team from Flytbase had contacted the antmedia with the license key yesterday.

**Reply by MBFLYT** [reply]
hey @USAMAWIZARD @Mohit-3196 We have been contacting Antmedia from yesterday, and we had sent the license as well, but haven't received a response and our production is down because of this issue. How can we accelerate this issue?

**Reply by yashtandon113** [reply]
@MBFLYT 
As I see the provided license key, you have two annual licenses, and both are being used on the servers.

**Reply by MBFLYT** [reply]
@yashtandon113 Thank you for the response, can you please reply this on the email so that other team members are also aware of this.

**Reply by MBFLYT** [reply]
Hello @yashtandon113 @Mohit-3196 @USAMAWIZARD @mustafaboleken,

We received your email yesterday regarding the license issue. According to your explanation, the license was suspended because we used it in the third instance. However, we had previously deleted one of the instances before starting to use the license in the third instance.

Could you please provide guidance on how we can resolve this issue? Any insights you can offer would be greatly appreciated.

Thank you

**Reply by Mohit-3196** [reply]
Hi @MBFLYT,
I just checked the entries and it should be fine now.
Can you please check now by entering the license key?

Thank you

**Reply by MBFLYT** [reply]
Hello @Mohit-3196 @yashtandon113 

We have restarted the server with the license key, but it is still throwing the same error.

**Reply by Mohit-3196** [reply]
Hi @MBFLYT ,
Are you available for a quick meeting?
If yes, let's meet at 12:30 IST and I can check this with you.

**Reply by MBFLYT** [reply]
Hello @Mohit-3196 

Yeah sure, I will be available at that time.

**Reply by Mohit-3196** [reply]
@MBFLYT,
I just sent you an invite on the support ticket.

**Reply by MBFLYT** [reply]
Thank you @Mohit-3196, We had received the invite.

**Reply by MBFLYT** [reply]
Hey @Mohit-3196 @USAMAWIZARD @yashtandon113 , 

The commands you shared via the latest email helped to solve this issue, thank you for your all support.

**Reply by Mohit-3196** [reply]
Thank you @MBFLYT,
Cheers
